cob_hardware_config/cob4-2/config/flexisoft.yaml

```diff
@@ -1,9 +1,9 @@
 host: 10.4.2.99
 port: "9100" # alternative syntax: !!str 9100
 odometry_timeout: 1.0
-threshold_linear_slow: 0.1
-threshold_linear_fast: 0.4
-threshold_angular_fast: 0.5
+threshold_linear_slow: 1.1
```

Comment:
which values do we need if we are at saturn? They've also been changed inside the msh account, from where we've always started our robot.launch. So @ipa-fmw are you sure we should revert this?